generate.sh

```diff
@@ -161,6 +161,15 @@ if [ $? -ne 0 ]; then
 fi
 print_log "Documentation Generated Successfully!\n"
 
+if [ -n "$APIDOCS_NAME" ] && [ -n "$APIDOCS_URL" ]; then
+  if [ "$APIDOCS_NAME" == "swagger" ]; then
+    git clone -q https://github.com/swagger-api/swagger-ui.git swagger >/dev/null 2>&1
```

Comment:
since we cloning stable release. can we have to copy of `dist` folder in Yaydoc only?. it'll increase build speed. if this suggestion is useful. please consider